pkg/sdk/client_extensions_unsafe.go

```diff
@@ -0,0 +1,65 @@
+package sdk
+
+import (
+	"context"
+	"database/sql"
+)
+
+func (c *Client) ExecUnsafe(ctx context.Context, sql string) (sql.Result, error) {
```

Comment:
If that is an extension maybe we shouldn't pollute the main client with such functions and we should just create a new client with the extension we want to add, what I mean by that:
```go
type UnsafeClient struct {
	*Client
}

func NewUnsafeClientFromDb(db *sql.DB) *UnsafeClient {
	return &UnsafeClient{
		Client: NewClientFromDB(db),
	}
}

func (c *UnsafeClient) UnsafeExec(ctx context.Context, sql string) (sql.Result, error) {
        // c (UnsafeClient) can call the same functions as regular client (even private ones, but they need to be in the same package)
	return c.exec(ctx, sql)
}
```

Reply:
Can we change it in the following PR?